src/main/java/com/tvtracker/enterprise/TvTrackerController.java

```diff
@@ -38,7 +38,6 @@
     @RequestMapping("/")
     public String index(){
         return "start";
-
     }
 
     /**
```

Comment:
Must've deleted this blank line out of habit, sorry but I guess it does make for cleaner code.
https://www.baeldung.com/java-clean-code